healthcareai/develop_supervised_model.py

```diff
@@ -13,6 +13,7 @@
 from sklearn.neighbors import KNeighborsClassifier
 from sklearn.linear_model import SGDClassifier
 
+import healthcareai.common.helpers
```

Comment:
Or, if you're only interested in using the `calculate_random_forest_mtry_hyperparameter` function, you can alias it in the import like this:
`from healthcareai.common.helpers import calculate_random_forest_mtry_hyperparameter as rf_helper`

Then you would use it like this in the code below:
`max_features = rf_helper(len(self.X_test.columns))`